### DISCUSSION TITLE
temperature_calibration and absolute offset

### DISCUSSION THREAD

**Discussion by Bascht74** [Koenkk/zigbee2mqtt#13197] [General]
Hi, can anybody explain to me how to use the temperature_calibration feature in z2m?

Do I have to enter the "real temperature" or something else? 
I am not quite sure what "absolute offset" means...

https://www.zigbee2mqtt.io/devices/9290019758.html#philips-9290019758
> temperature_calibration: Calibrates the temperature value (absolute offset), takes into effect on next report of device. The value must be a number.

**Comment by sjorge** [comment]
You enter the difference.

Some examples:
- reported=30,actual=28.5 --> you enter `-1.5`
- reported=18,actual=20 --> you enter `2`

**Reply by Bascht74** [reply]
I will try!

**Comment by kapabel** [comment]
I have tried this but it doesn't work. My Aqara water leak sensor does also have a device temperature exposed but this one is constantly 5° wrong. I entered "-5" but it looks like it isn't taken into account. I have also tried to restart zigbee2mqtt but no change. What's wrong?

**Reply by lux73** [reply]
works as expected. you have to wait until your Sensor is updating the Value
